### PR TITLE
feat: refactor profile xml

### PR DESCRIPTION
**Pull request by rjmah1**
#### Summary of changes/features added:
- added comments and refactored layout.

#### Screenshots of UI changes (update with each new commit): 
- No new change

## Pull Request Checklist
- [ ] Unit test written?
- [ ] (**For later in project**) Integration tests written?
- [x] Java docs written?

### CODE REVIEW

File: client/app/src/main/java/com/example/criengine/UserProfileActivity.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import android.os.Bundle;
+public class UserProfileActivity extends ProfileActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        setContentView(R.layout.activity_user_profile);
+        super.onCreate(savedInstanceState);
+
+        // new UI components
```
Comment (AdityaHarvi) [inline client/app/src/main/java/com/example/criengine/UserProfileActivity.java:11]: I'm assuming these are yet to be completed activities?

Also side not: Remember to PR titles to be in the form of "feat: <>" or "fix: <>" etc. Its more descriptive as to what is going on and will be easier to find if we need to later on

Comment (rjmah1) [reply client/app/src/main/java/com/example/criengine/UserProfileActivity.java:11]: yea i'm not sure if i'd be able to finish that by today (need to work on 379 a bit) but it's some placeholder for when I get to it next week

Comment (AdityaHarvi) [reply client/app/src/main/java/com/example/criengine/UserProfileActivity.java:11]: Sounds good.
We did agree to only submit PR's when they were in their final working state so I'll assign the WIP tag for this.

Everything else is just a refactor so I suppose its nothing urgent.

Comment (rjmah1) [reply client/app/src/main/java/com/example/criengine/UserProfileActivity.java:11]: this PR is done, and is in a working state (none of that code is run)

Comment (AdityaHarvi) [reply client/app/src/main/java/com/example/criengine/UserProfileActivity.java:11]: i supoooose soo...

I guess in the future we should avoid adding in files that don't relate to the purpose of the PR 🤷 (the empty activities)

**Review by AdityaHarvi — APPROVED** [review]

File: client/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
     <string name="app_name">CriEngine</string>
+    <string name="lorem_ipsum">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris consectetur rutrum enim, finibus consectetur ex pharetra a. Donec tellus massa, iaculis facilisis laoreet et, condimentum vitae mauris. Fusce et neque eu dolor commodo sollicitudin. Duis vel viverra tellus. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vivamus est tellus, molestie sed eros in, bibendum pulvinar lectus. Vestibulum fringilla euismod lorem in imperdiet. Vivamus venenatis ante lobortis mollis mattis. Curabitur pulvinar elit at venenatis porta. Etiam et scelerisque justo, vitae dignissim leo. Donec elementum mi sit amet ante aliquet varius. Proin pretium lacus a nibh sodales aliquam.</string>
+
+    <!-- Profile Component -->
     <string name="user_profile_text">%1$s\'s Profile</string>
     <string name="bio_header">Bio</string>
     <string name="contact_info">Contact Information</string>
-    <string name="phone_text">Phone:</string>
-    <string name="address_text">Address:</string>
+    <string name="phone_header">Phone:</string>
```
Comment (SamLojpur) [inline client/app/src/main/res/values/strings.xml:9]: I know this is soooo stupid but can we put a space after the colon for these two? its a little bit cramped looking
image

**Review by SamLojpur — APPROVED** [review]
LGTM, maybe look at this later when you have the chance

**Pull request merged**